src/app/shared/models/rumble.d.ts

```diff
@@ -62,20 +62,13 @@ export interface RumbleSchema {
 }
 export interface Unit {
   ability: [Ability, Ability, Ability, Ability, Ability];
-  global?: Unit;
```

Comment:
I think these need to stay....I might need to check to see if I put them in the schema.json, if not, they need to be there. This is for when units have different stats between the global and JP versions.

Reply:
Don't worry about them for now, I can add them back in if necessary. We don't do anything with it yet, this was me planning for the future.

Reply:
yeah i checked on the raw json and none of the objects there had these properties.. so to to make it "match the auto generated types from json" it made sense to delete

Reply:
I just checked, it looks like they're in the optc-db version of the schema. I need to set it up to auto pull and auto regen rumble.d.ts